src/pages/Settings/Settings.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react'
+import {
+  Card,
+  CardContent,
+  Typography,
+  Grid,
+  Switch,
+  FormGroup,
+  FormControlLabel,
+  Button,
+} from '@material-ui/core'
+
+import { useStyles } from './styles'
+
+const Settings: FC = () => {
+  const classes = useStyles()
+
+  const [checked, setChecked] = React.useState(false)
```

Comment:
It's nice to name booleans with one naming convention. I started naming them with `is` prefix everywhere, so I guess you can try that too.

Reply:
```suggestion
  const [isChecked, setIsChecked] = React.useState(false)
```

Reply:
Also, you can import `useState`
```suggestion
  const [isChecked, setIsChecked] = useState(false)
```
```ts
import React, { FC, useState } from 'react'
```